src/plugins/data_source_management/public/components/popover_button/popover_button.scss

```diff
@@ -0,0 +1,66 @@
+.dataSourceMenuPopoverButton {
+  position: relative;
+  background-color: $euiHeaderBackgroundColor;
+  border: 1px solid $euiColorMediumShade;
```

Comment:
Any reason we're not using a button border color for this? I'm guessing this is pre-existing?

Reply:
This matches the UX but would be easy to change.